src/RobotInterface/src/Helper.cpp

```diff
@@ -230,6 +239,12 @@ bool RobotInterface::configureRobot(const yarp::os::Searchable& config)
         return false;
     }
 
+    if(!m_robotDevice.view(m_InteractionInterface) || !m_InteractionInterface)
+        {
+             yError() << "[configureRobot] Cannot obtain IInteractionMode interface";
+            return false;
+         }
```

Comment:
Can you follow the [indentation style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) of the file?